### PR TITLE
Update FreeDesktop.xml

### PR DESCRIPTION
**Pull request by J0WI**
#15750

### CODE REVIEW

File: src/chrome/content/rules/FreeDesktop.xml [hunk 1]
```diff
@@ -51,24 +47,10 @@
 	<target host="xcb.freedesktop.org" />
 	<target host="xorg.freedesktop.org" />
 
-	<!--	Complications:
-				-->
-	<target host="download.freedesktop.org" />
-	<target host="sitewranglers.freedesktop.org" />
-
-		<test url="http://people.freedesktop.org/~airlied/" />
```
Comment (J0WI) [inline src/chrome/content/rules/FreeDesktop.xml]: Please keep this test url because there is no content on `http://people.freedesktop.org/`.

**Comment by J0WI** [comment]
@cschanaj could you review our changes?

File: src/chrome/content/rules/FreeDesktop.xml [hunk 2]
```diff
@@ -1,96 +1,58 @@
...
 	<target host="piglit.freedesktop.org" />
+	<target host="planet.freedesktop.org" />
 	<target host="pm-utils.freedesktop.org" />
 	<target host="poppler.freedesktop.org" />
 	<target host="portland.freedesktop.org" />
 	<target host="secure.freedesktop.org" />
+	<target host="sitewranglers.freedesktop.org" />
 	<target host="specifications.freedesktop.org" />
 	<target host="standards.freedesktop.org" />
 	<target host="swfdec.freedesktop.org" />
```
Comment (cschanaj) [inline src/chrome/content/rules/FreeDesktop.xml]: I get mismatched certificate for `swfdec.freedesktop.org`

File: src/chrome/content/rules/FreeDesktop.xml [hunk 3]
```diff
@@ -1,96 +1,58 @@
...
 
+	<securecookie host=".+" name=".+" />
 
-	<securecookie host="^bug" name=".+" />
-
-
-	<!--	Redirects as so:
-				-->
 	<rule from="^http://(?:download|sitewranglers)\.freedesktop\.org/"
 		to="https://fontconfig.freedesktop.org/" />
```
Comment (cschanaj) [inline src/chrome/content/rules/FreeDesktop.xml]: For both `download` and `sitewranglers` I get `302` to `https://www.freedesktop.org/wiki/` . The content is also different from `fontconfig`, this rule might need a rewrite.

Comment (J0WI) [reply src/chrome/content/rules/FreeDesktop.xml]: I'm getting a server side redirect to https. Removing this rule completely now.

**Pull request merged**